core/templates/dev/head/domain/exploration/InteractionObjectFactory.js

```diff
@@ -51,7 +52,7 @@ oppia.factory('InteractionObjectFactory', [
           return hint.toBackendDict();
         }),
         id: this.id,
-        solution: {}
```

Comment:
Yes, default solutions we're {} during our last migration. So, we'll have to migrate again (to None)?